umbraco/Umbraco-CMS
# Remove macros

### Prerequisites


### Description

This PR removes macros! 🥳 

### Testing this PR

Nothing should really change at API level. At all. The Management API does not expose nor concern itself with macros.

Test this by:

1. Verify that the `cmsMacro` and `cmsMacroProperty` tables are removed from the database once the site starts (on an existing V13/V14 database).
2. Get a partial view snippet - this should _not_ fail.
3. Smoke test the Management API.
4. Run the installer (or install unattended) and verify that the `cmsMacro` and `cmsMacroProperty` tables are not created at install time.
